tests/test_desktopcontrol.py

```diff
@@ -78,6 +70,10 @@ def setUp(self):
     def tearDown(self):
         if os.path.exists(GlobalConfig.image_logging_destination):
             shutil.rmtree(GlobalConfig.image_logging_destination)
+        for desktop in self.backends:
+            # disconnect any vncdotool backend
+            if isinstance(desktop, VNCDoToolDesktopControl):
+                desktop._backend_obj.disconnect()
```

Comment:
Maybe later on we should think about some sort of closure of DC backends. In the same way we initiate screens we might think about a method offering closing screens. Just a thought though.

Reply:
Agreed, perhaps adding some context capabilities would work nicely.